### ISSUE TITLE
Undefined method `each`

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss-labs/rialto-etl#134]
```
exe/transform call StanfordPeople -i data/researchers.ndj > researchers.sparql
2018-09-20T13:44:53-05:00  INFO    Indexer with 1 processing threads, reader: Rialto::Etl::Readers::NDJsonReader and writer: Rialto::Etl::Writers::SparqlStatementWriter
2018-09-20T13:45:01-05:00 ERROR Unexpected error on record id `` at file position 1022
    while executing (to_field http://vivoweb.org/ontology/core#relatedBy at lib/rialto/etl/configs/stanford_people_to_sparql_statements.rb:180)
    Exception: NoMethodError: undefined method `each' for nil:NilClass
    lib/rialto/etl/configs/stanford_people_to_sparql_statements.rb:184:in `block (2 levels) in load_config_file'
```

**Comment by jcoyne** [comment]
It appears that `titles` is not always present in the response.

**Comment by jcoyne** [comment]
Fixed by #135
